README.md

```diff
@@ -123,17 +123,22 @@ use the `artifact-name` parameter:
+| Parameter                        | Description                                                                                                                                             | Default                          |
+| -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------- |
+| `path`                           | A path on the filesystem to scan. This is mutually exclusive to `file` and `image`.                                                                     | \<current directory>             |
+| `file`                           | A file on the filesystem to scan. This is mutually exclusive to `path` and `image`.                                                                     |                                  |
+| `image`                          | A container image to scan. This is mutually exclusive to `path` and `file`. See [Scan a container image](#scan-a-container-image) for more information. |                                  |
+| `registry-username`              | The registry username to use when authenticating to an external registry                                                                                |                                  |
+| `registry-password`              | The registry password to use when authenticating to an external registry                                                                                |                                  |
+| `artifact-name`                  | The name to use for the generated SBOM artifact. See: [Naming the SBOM output](#naming-the-sbom-output)                                                 | `sbom-<job>-<step-id>.spdx.json` |
+| `output-file`                    | The location to output a resulting SBOM                                                                                                                 |                                  |
+| `format`                         | The SBOM format to export. One of: `spdx`, `spdx-json`, `cyclonedx`, `cyclonedx-json`                                                                   | `spdx-json`                      |
+| `dependency-snapshot`            | Whether to upload the SBOM to the GitHub Dependency submission API                                                                                      | `false`                          |
+| `upload-artifact`                | Upload artifact to workflow                                                                                                                             | `true`                           |
+| `upload-artifact-retention-days` | Retention policy for uploaded artifact to workflow.                                                                                                     |                                  |
```

Comment:
I wonder if the name `upload-artifact-retention` would be preferable with documentation indicating it is in days?

Reply:
I could remove `-days`, i think its fine.

Reply:
Done